util/file.py

```diff
@@ -241,6 +241,9 @@ def extract_tarball(tarfile, out_dir=None, threads=None, compression='auto', pip
         elif compression == 'none':
             decompressor = ['cat']
         untar_cmd = ['tar', '-C', out_dir, '-x']
+        if os.getuid() == 0:
+            log.debug("tar is running as root. Assuming that this is GNU tar, appending --no-same-owner")
+            untar_cmd.append('--no-same-owner')
```

Comment:
Should we have a [conditional](https://github.com/broadinstitute/viral-ngs/blob/f8a8a699748a0628721799d00867503a72544098/tools/megan.py#L84) to only append this to `tar` in the non-Mac case?

Reply:
Ok, added a new conditional check. I didn't want to check for MacOSX though, I decided to check for GNU tar, since that's what the behavior is specific to (just in case someone wants to run viral-ngs as root on a FreeBSD system).